app/views/categories/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<nav>
+  <div><h2>Categories</h2></div>
+  <%= button_to 'Sign Out', destroy_user_session_path, method: :delete %>
+</nav>
+<section class="main">
+  <% @categories.each do |category| %>
+    <section class="category">
+      <div>
+        <img src="<%= category.icon %>">
+        <%= link_to category_procurements_path(category) do %>
+          <%= category.name %>
+        <% end %>
```

Comment:
| @Wineshuga Design |
|-|
| ![image](https://github.com/Wineshuga/buddy_budget/assets/93039359/a6a0ceaf-d32d-4503-96dd-5464b863d49f) |

| Microverse Design |
|-|
| ![image](https://github.com/Wineshuga/buddy_budget/assets/93039359/9d63b65c-8ce4-41b4-8bf8-96c3b41ffe14) |

- [ ] It would be nice if you make your categories card look similar by fixing the color of the font, font-family and by adding the date 👍